stock_putaway_product/i18n/stock_putaway_byproduct.pot

```diff
@@ -0,0 +1,88 @@
+# Translation of Odoo Server.
```

Comment:
Remove this file, as it's automatically generated when committing before pushing to Transifex, and it can generate confusions.